galette/templates/default/pages/preferences.html.twig

```diff
@@ -921,8 +921,7 @@
                 $('#btnmail').on('click', function(e) {
                     e.preventDefault();
                     var _this = $(this);
-                    var _value = $('#pref_email_newadh').val();
-                    var _input = '<div class="ui input"><input type="text" name="email_adress" id="email_adress" value="' + _value + '"/></div>';
+                    var _input = '<div class="ui input"><input type="text" name="email_adress" id="email_adress" value=""/></div>';
```

Comment:
An alternative would be to use something like:
```
_value.split(',', 1)[0]
```

Reply:
I don't know why, but doing that, the modal is never displayed and the test message sent directly. 

When the "members administrator's emails" field is not filled, it is not possible to choose a custom recipient in the modal and sending the test message fails.

Reply:
OK, if all is OK with the field empty; it's enough for me ;) thanks!